addon/templates/components/validated-input.hbs

```diff
@@ -38,18 +38,19 @@
   {{#if (eq type "select")}}
 
     {{one-way-select
-      value            = (or value (get model name))
-      options          = options
-      optionLabelPath  = optionLabelPath
-      optionValuePath  = optionValuePath
-      optionTargetPath = optionTargetPath
-      name             = name
-      class            = config.css.control
-      update           = (action "update")
-      focusOut         = (action "setDirty")
-      includeBlank     = includeBlank
-      disabled         = disabled
-      multiple         = multiple
+    value               = (or value (get model name))
+    options             = options
+    optionLabelPath     = optionLabelPath
+    optionValuePath     = optionValuePath
+    optionTargetPath    = optionTargetPath
+    name                = name
+    class               = "form-control"
+    update              = (action "update")
+    focusOut            = (action "setDirty")
+    prompt              = (if includeBlank (or includeBlank null))
```

Comment:
I just realized that we were supporting `includeBlank` already. Since `prompt` and includeBlank are functionally equivalent, it might make most sense to just keep `includeBlank = includeBlank`, and a little comment ("We're not supporting `prompt`, please just use `includeBlank` instead") to the README. As soon as the issue in one-way-controls is fixed, we could add `prompt`.
In case I'm missing something here it might make sense if you could shed some more light on your concrete use-case for this!